docs/20-glossary.rst

```diff
@@ -47,7 +47,7 @@ help others.
   `Gulp <http://gulpjs.com/>`_
```

Comment:
Do we still need info about gulp, can't we just remove it completely instead?

Reply:
I have now removed texts about gulp. Replaced gulp serve with yarn start where I found possible.